packages/flame/src/Alert/Alert.tsx

```diff
@@ -90,11 +95,19 @@ const Alert: React.FunctionComponent<AlertProps & SpaceProps> = ({
         )}
         <Box flex="1">
           {title && (
-            <Text color="textHeading" fontWeight="bold" fontSize="text" mt={0} mr={0} mb={1} ml={0}>
+            <Text
```

Comment:
line 92: aligning the icon vertically like this is not sustainable. Can we `flex` it?